src/main/java/Bad.java

```diff
@@ -0,0 +1,22 @@
+public class Bad {
+
+    public Bad() {
+        if (false) {
+            System.out.println("");
+        }
+    }
+
+    public int foo() {
+        String foo = null;
+
+        return Integer.getInteger(foo);
+    }
+
+    public class BadInner {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vvisteria/danger-spotbugs-sample/blob/c85e89d787406204ea33050153f40192b2ef1c41/src/main/java/Bad.java#L15"></span>Should Bad$BadInner be a <em>static</em> inner class?</td>
    </tr>
  </tbody>
</table>